dnf/base.py

```diff
@@ -2536,6 +2536,14 @@ def _raise_package_not_installed_error(self, pkg_spec, forms, reponame):
             msg = _('All matches were filtered out by exclude filtering for argument')
         raise dnf.exceptions.PackagesNotInstalledError(msg, pkg_spec)
 
+    def setup_loggers(self, conf=None):
```

Comment:
Is there any advantage to use an external conf for class function that could be initialized by external config object?

Reply:
No, there isn't. It's a leftover from former dead-end code. I'll remove it.